nifi-assembly/NOTICE

```diff
@@ -818,6 +818,15 @@ The following binary components are provided under the Apache Software License v
        The code for the t-digest was originally authored by Ted Dunning
        A number of small but very helpful changes have been contributed by Adrien Grand (https://github.com/jpountz)
 
+    (ASLv2) subethasmtp
+       The following NOTICE information applies:
+
+       Copyright (C) 2006-2007 SubEthaMail.org
```

Comment:
This project is Licensed under ASLv2[1] without any notice[2] so it can be removed from our NOTICE file. 

[1] https://github.com/voodoodyne/subethasmtp/blob/3.1.7/LICENSE.txt
[2] https://github.com/voodoodyne/subethasmtp/tree/3.1.7